packages/merge-styles/src/IRawStyleBase.ts

```diff
@@ -1284,6 +1310,40 @@ export interface IRawStyleBase extends IRawFontStyle {
    */
   paddingLeft?: ICSSRule | ICSSPixelUnitRule;
 
+  /**
+   * The padding-bottom CSS property of an element sets the padding space required on
+   * the bottom of an element. The padding area is the space between the content of the
+   * element and its border. Contrary to margin-bottom values, negative values of
+   * padding-bottom are invalid.
+   */
+  paddingBottom?: ICSSRule | ICSSPixelUnitRule;
+
+  /**
+   * The padding-inline-end CSS property defines the logical inline end padding of an element,
+   * which maps to a physical padding depending on the element's writing mode, directionality,
+   * and text orientation. It corresponds to the padding-top, padding-right, padding-bottom,
+   * or padding-left property depending on the values defined for writing-mode, direction,
+   * and text-orientation.
+   */
+  paddingInlineEnd?: ICSSRule | ICSSPixelUnitRule;
+
+  /**
+   * The padding-inline-start CSS property defines the logical inline start padding of
+   * an element, which maps to a physical padding depending on the element's writing mode,
+   * directionality, and text orientation. It corresponds to the padding-top, padding-right,
+   * padding-bottom, or padding-left property depending on the values defined for writing-mode,
+   * direction, and text-orientation.
+   */
+  paddingInlineStart?: ICSSRule | ICSSPixelUnitRule;
+
+  /**
+   * The padding-block-end CSS property defines the logical block end padding
```

Comment:
Looks like this is the comment for paddingBlockEnd

Reply:
Copy paste error after re-sorting the props.
Fixed now.